streams/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/rule/sql/SqlScript.java

```diff
@@ -53,6 +50,9 @@
     private final List<Schema.Field> stormSqlFields;
     private final List<String> projectedFields;
     private final List<String> outputFields;
+    private static final Schema.Field DUMMY_FIELD = Schema.Field.of("dummy", Schema.Type.INTEGER);
```

Comment:
When there are no references to input fields, we add a dummy field so that the
rule table is created with the dummy field and the values can be processed by storm-sql.
e.g. SELECT RAND() from inputStream results in 
       CREATE EXTERNAL TABLE RULETABLE (dummy INTEGER)